### ISSUE TITLE
Edge doesn't display blog posts

### ISSUE THREAD

**Issue by Croydon** [inexorgame/inexor-core#549]
Edge just displays this:

![edge](https://user-images.githubusercontent.com/1593194/34636355-6ed1c45c-f29f-11e7-85e9-dbd80a3d8fc8.PNG)

1. Write a comment here to let us know that you are interested in working on this issue
2. Investige why Edge can't load and display our posts like Firefox, Chrome and (assumigly) Safari (can somebody test Safari?)
     * the source code can be found here: https://github.com/inexorgame/site
3. Create a pull request in the site repository with your fix
4. Wait for feedback and if neccessary make changes
5. If approved we are going to merge your fix 

If you have questions feel free to ask anything here, we are glad to help :)

**Comment by zahiryahya1** [comment]
I am interest in working on this issue

**Comment by Croydon** [comment]
@zahiryahya1 Awesome! Let us know if you have any questions 😄

**Comment by zahiryahya1** [comment]
I narrowed down the problem to parseBlogArray(arr) function in Blog.vue. It seems to have something to do with parsing the 'path' object to get the date and title using regular expressions. The way I got around it was to convert 'path' to a string and get the sub-strings corresponding to date and title. My solution assumes  that 'path' string is of the form 'post/2018/01-27-title.md'.  Am I OK to make this assumption?
